sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/PartitionManager.cs

```diff
@@ -171,6 +171,8 @@ async Task RunAsync()
                         ProcessorEventSource.Log.EventProcessorHostWarning(this.host.HostName, retryMessage, ex.ToString());
                     }
 
+                    this.host.EventProcessorOptions.NotifyOfException(this.host.HostName, partitionId, ex, action);
```

Comment:
nit:  The options seems a weird place to have this call, but since it was already there... probably not worth discussing further.

Reply:
Agreed, it better be on the host instead.

---
In reply to: [334483484](https://github.com/Azure/azure-sdk-for-net/pull/8088#discussion_r334483484) [](ancestors = 334483484)